style.css

```diff
@@ -1,448 +1,1106 @@
+  }
+
+  .card2001,
+  .card4001 {
+    order: 2;
+  }
+
+  ul {
+    display: flex;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    list-style: none;
+    color: #344563;
+  }
+
+  h3 {
+    color: #344563;
+    margin-right: 7px;
+  }
+
+  p {
+    width: 295px;
+    height: 76px;
+    margin: 8px 7px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .lang {
+    float: left;
+    width: 100px;
+    height: 16px;
+    list-style: none;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  h5 {
+    background-color: #ebebff;
+    padding: 4px 12px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    border-radius: 8px;
+  }
+
+  .button {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    box-sizing: border-box;
+    width: 150px;
+    padding: 12px 3px;
+    background: #fff;
+    border: 1px solid #435c9d;
+    border-radius: 14px;
+    margin-top: 3rem;
+    transition: color 0.5s ease;
+  }
+
+  button:hover {
+    color: crimson;
+  }
+
+  .card1 {
+    display: none;
+  }
+
+  .c002 > h2,
+  h3 {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .card001 {
+    margin-top: 0%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+    width: 100%;
+  }
+
+  .c002 {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    font-weight: 400;
+    font-size: 16px;
+    padding-left: 3%;
+    line-height: 24px;
+  }
+
+  .card3 {
+    display: none;
+  }
+
+  .cards1 {
+    order: 1;
+  }
+
+  .card2 {
+    order: 4;
+  }
+
+  .swapcard3 {
+    order: 3;
+  }
+
+  .card4 {
+    order: 2;
+  }
+
+  .about {
+    margin-top: 4rem;
+    width: 100%;
+    background-color: white;
+    height: auto;
+    border-top-right-radius: 5rem;
+  }
+
+  .intro {
+    width: 100%;
+    height: auto;
+    color: #344563;
+    font-size: 16px;
+    font-family: "Poppins", sans-serif;
+    line-height: 24px;
+    font-style: normal;
+    font-weight: 400;
+  }
+
+  #about {
+    width: 100%;
+    padding-top: 2rem;
+    display: grid;
+    grid-template-columns: 1fr 1.7fr;
+  }
+
+  #about001 {
+    padding-left: 15%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  #about002 {
+    display: flex;
+    margin-left: 3%;
+    flex-direction: column;
+  }
```

Comment:
- [ ] I would kindly suggest you give this section a `justify content` to be centered which will give space to the left and right just as it is in the Figma design.

- [ ] Also, the content should be 40% width for the left and 60% width for the right but yours is vis-a-vis. Kindly correct that too, with this you would have structured your about section just exactly like what we have in the Figma template.

Your design 👇 

![bob abt](https://user-images.githubusercontent.com/78634841/208093441-ea79c1d8-f124-4776-8618-2716b61b98b4.png)

Figma design 👇 

![abut real](https://user-images.githubusercontent.com/78634841/208093648-43ba7d80-4459-48c6-9001-8e1ab9ee758d.png)